packages/api/models/granules.js

```diff
@@ -482,6 +483,19 @@ class Granule extends Manager {
     }
     return executionDescription;
   }
+
```

Comment:
When granule is not in dynamoDB, unpublish has error:
2023-04-18T15:46:33.756Z 489c96c9-fec7-411c-84b4-eac0df058c41 ERROR
{ "error": { "name": "RecordDoesNotExist", "message": "No record found for {\"granuleId\":\"MOD09GQ.A7614271.S4rfYb.006.4748017559580\"} in jl-tf-GranulesTable", "stack": [ "RecordDoesNotExist: No record found for {\"granuleId\":\"MOD09GQ.A7614271.S4rfYb.006.4748017559580\"} in jl-tf-GranulesTable", " at Object.get (/var/task/webpack:/src/DynamoDb.ts:63:9)", " at processTicksAndRejections (node:internal/process/task_queues:96:5)", " at Granule.get (/var/task/webpack:/models/base.js:228:12)", " at Granule.get (/var/task/webpack:/models/granules.js:88:29)", " at Granule.update (/var/task/webpack:/models/base.js:372:25)", " at /var/task/webpack:/lib/granule-remove-from-cmr.js:95:29" ] }, "level": "error", "message": "No record found for {\"granuleId\":\"MOD09GQ.A7614271.S4rfYb.006.4748017559580\"} in jl-tf-GranulesTable", "sender": "@cumulus/api", "timestamp": "2023-04-18T15:46:33.756Z" }